src/resolvers/element-ui.ts

```diff
@@ -0,0 +1,73 @@
+import {
+  ComponentResolver,
+  ImportInfo,
+} from '../types'
+import { kebabCase } from '../utils'
+
+export interface ElementUiResolverOptions {
+  /**
+   * import style along with components
+   *
+   * @default true
+   */
+  importStyle?: boolean
```

Comment:
```suggestion
  importStyle?: boolean | 'css' | 'less'
```

Instead of three options, would you think this could be better?

Reply:
en , it is better, i will change it later, and submit a new pr